client/my-sites/earn/memberships/index.jsx

```diff
@@ -697,6 +729,7 @@ const mapStateToProps = ( state ) => {
 			siteHasFeature( state, siteId, FEATURE_DONATIONS ) ||
 			siteHasFeature( state, siteId, FEATURE_RECURRING_PAYMENTS ),
 		isJetpack: isJetpackSite( state, siteId ),
+		source: window.location.hash === '#add-newsletter-payment-plan' ? 'earn-newsletter' : 'calypso',
```

Comment:
```suggestion
		source: window.location.hash === ADD_NEWSLETTER_PAYMENT_PLAN_HASH ? 'earn-newsletter' : 'calypso',
```

Reply:
Fixed in [Replace magic string with const](https://github.com/Automattic/wp-calypso/pull/74146/commits/adce253dadc976f5803a918ed9aae4a0b2d2c80d)